clients/src/test/java/org/apache/kafka/clients/telemetry/TelemetryManagementInterfaceTest.java

```diff
@@ -56,6 +69,55 @@ public void testNoneCompressedSerialization() throws IOException {
         testSerialization(CompressionType.NONE);
     }
 
+    @Test
+    public void testValidateMetricNames() {
+        // empty metric names
+        assertTrue(TelemetryManagementInterface.validateMetricNames(Arrays.asList()).isEmpty());
+        assertTrue(TelemetryManagementInterface.validateMetricNames(null).isEmpty());
+    }
+
+    @Test
+    public void testValidateAcceptedCompressionTypes() {
+        // invalid compression types
+        assertTrue(TelemetryManagementInterface.validateAcceptedCompressionTypes(null).isEmpty());
+        assertTrue(TelemetryManagementInterface.validateAcceptedCompressionTypes(new ArrayList<>()).isEmpty());
```

Comment:
Can we use `Collections.emptyList()` here, instead?

Reply:
Can we use `assertEquals()` and pass in the sizes, instead? That would add a little more information to the error message.